plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -131,7 +141,7 @@ public static HiveMetastore cachingHiveMetastore(HiveMetastore delegate, Executo
 
     public static HiveMetastore cachingHiveMetastore(HiveMetastore delegate, Executor executor, Duration cacheTtl, Optional<Duration> refreshInterval, long maximumSize)
```

Comment:
make it return `CachingHiveMatestore`

Reply:
Same as https://github.com/trinodb/trino/pull/10251#discussion_r768621426